### PR TITLE
L diversity

### PR DESCRIPTION
**Pull request by JulianSagen**
Added Ldiversity anonymization model classes
100% test coverage!

### CODE REVIEW

File: pyaaas/models/privacy_models.py [hunk 1]
```diff
@@ -2,35 +2,64 @@
...
+    def __iter__(self):
+        return iter(self._internal_dict)
+
+    @property
     def name(self) -> str:
-        raise NotImplementedError()
+        return self._anonymity_name
 
+    def __str__(self):
+        return self._print_messqge
```
Comment (sonhal) [inline pyaaas/models/privacy_models.py]: typo?

File: pyaaas/models/privacy_models.py [hunk 2]
```diff
@@ -2,35 +2,64 @@
 from abc import ABC, abstractproperty
 
 
-class PrivacyModel(ABC):
+class PrivacyModel(ABC, Mapping):
     """ ABC for ARX Privacy Models"""
```
Comment (sonhal) [inline pyaaas/models/privacy_models.py:6]: [Forslag] Endre til å instansiere self._internal_dict i PrivacyModel og så la subklasser gjennbruke den.

File: tests/pyaaas/models/test_privacy_models_LDiversity.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from pyaaas.models.privacy_models import LDiversityDistinct, LDiversityGrassbergerEntropy, LDiversityShannonEntropy, LDiversityRecursive
+from pyaaas.aaas import AaaS
+import unittest
+
+class PrivacyModelsTest(unittest.TestCase):
```
Comment (sonhal) [inline tests/pyaaas/models/test_privacy_models_LDiversity.py]: Burde ikke denne hete LDiversityDinstinctTest?

**Review by sonhal — CHANGES_REQUESTED** [review]

File: pyaaas/models/privacy_models.py (hunk 1 above)
Comment (JulianSagen) [reply pyaaas/models/privacy_models.py]: Solved

File: tests/pyaaas/models/test_privacy_models_LDiversity.py (hunk 3 above)
Comment (JulianSagen) [reply tests/pyaaas/models/test_privacy_models_LDiversity.py]: Split tests into multiple files, and refactored the names

File: tests/pyaaas/models/test_anonymize_payload.py [hunk 4]
```diff
@@ -28,7 +28,23 @@ def test_set_data__add_dataframe(self):
         self.assertIsInstance(self.test_payload.data, str)
```
Comment (sonhal) [inline tests/pyaaas/models/test_anonymize_payload.py:28]: remove

File: tests/pyaaas/models/test_privacy_models.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
+from pyaaas.models.privacy_models import LDiversityDistinct
+import unittest
+
+class PrivacyModelsTest(unittest.TestCase):
```
Comment (sonhal) [inline tests/pyaaas/models/test_privacy_models.py]: What are we really testing?

**Review by sonhal — CHANGES_REQUESTED** [review]

**Review by sonhal — APPROVED** [review]
lgtm

**Pull request merged**